crates/autopilot/src/arguments.rs

```diff
@@ -82,13 +82,7 @@ pub struct Arguments {
 
     /// Which estimators to use to estimate token prices in terms of the chain's
     /// native token.
-    #[clap(
-        long,
-        env,
-        default_value = "Baseline",
-        value_enum,
-        use_value_delimiter = true
-    )]
+    #[clap(long, env, value_enum, use_value_delimiter = true)]
     pub native_price_estimators: Vec<shared::price_estimation::PriceEstimatorType>,
```

Comment:
There shouldn't be solver addresses associated with native price queries. Rewards are just tied to actual quotes and not the native token prices used for the estimates.

Reply:
Rewards are also not tied to limit orders. My plan was to have a consistent data model (i.e. store the address for all orders) and then let analytics simply filter out all orders which should not be rewarded.

Reply:
The consequence of this change is that `cargo run -p orderbook` no longer works by default. You have to remember to specify both `--price-estimator` and `--native-price-estimator`.

I agree with the "My plan was to have a consistent data model" argument, but maybe to make testing easier we can have `Baseline|0x0000000000000000000000000000000000000000` as the default argument value for the flags?

Reply:
Oh nice. I actually like the fix of defaulting to the 0 address.